Source/Pages/Gallery/LibraryMediaManager.swift

```diff
@@ -197,7 +199,13 @@ public class LibraryMediaManager {
 
                 // 5. Configuring export session
                 let videoIsCropped = cropRect.size.width < abs(videoSize.width) || cropRect.size.height < abs(videoSize.height)
-                let presetName = compressionTypeOverride ?? (videoIsCropped || videoIsTrimmed || videoIsRotated || (shouldMute && videoNeedsProcessing) ? YPConfig.video.compression : AVAssetExportPresetPassthrough)
+
+                var presetName: String = ""
+                if !YPImagePickerConfiguration.shared.library.allowPhotoAndVideoSelection {
+                    presetName = compressionTypeOverride ?? (videoIsCropped || videoIsTrimmed || videoIsRotated || (shouldMute && videoNeedsProcessing) ? YPConfig.video.compression : AVAssetExportPresetPassthrough)
+                } else {
+                    presetName = AVAssetExportPresetHighestQuality
```

Comment:
Where did `AVAssetExportPresetHighestQuality` come from? Seems unrelated to what was here before.

Reply:
Yeah it was on the WIP branch but tested without this change and I didn't see any issues with the multi video selection. Removed it here: https://github.com/rewardStyle/YPImagePicker/pull/27/commits/d11a88a5c675d3787e42e42e08037e91bba1cb99